Toxygates/src/panomicon/ScalatraJSONServlet.scala

```diff
@@ -398,7 +448,7 @@ class ScalatraJSONServlet(scontext: ServletContext) extends ScalatraServlet
   }
 
   get("/batch") {
-    verifyRole("admin")
+    verifyRole(ADMIN_ROLE)
```

Comment:
Changed this magic string to a constant throughout the file, as that seemed cleaner.